prestoadmin/slider/config.py

```diff
@@ -135,17 +149,23 @@ def write(conf):
 
 
 def get_conf_if_missing():
+    """ Loads the configuration if it hasn't already been loaded. Returns True
+    if the config was loaded for the first time, and False otherwise.
+    """
     with settings(parallel=False):
```

Comment:
Boy did this open a can of worms. It's fixed with the config load changes that went in last week.